flixel/input/touch/FlxTouch.hx

```diff
@@ -75,11 +77,19 @@ class FlxTouch extends FlxPointer implements IFlxDestroyable implements IFlxInpu
 		{
 			justPressedPosition.set(screenX, screenY);
 			justPressedTimeInTicks = FlxG.game.ticks;
+			#if FLX_POINTER_INPUT 
+			FlxG.swipes.push(_currentSwipe = new FlxSwipe(touchPointID, justPressedPosition, justPressedTimeInTicks)); 
+			#end
 		}
 		#if FLX_POINTER_INPUT
+		else if (pressed)
+		{
+			_currentSwipe.path.addPoint(getScreenPosition());
```

Comment:
So the path will contain a point for each frame that the swipe was active for? I wonder if that isn't a bit much.